frontend/src/stores/PlayersStore.ts

```diff
@@ -0,0 +1,31 @@
+import { makeAutoObservable } from 'mobx';
+import Technologies from '../constants/technologies';
+import Player from '../model/Player';
+import { RootStore } from './RootStore';
+
+class PlayersStore {
+  players: Player[];
+  rootStore: RootStore;
+
+  constructor(rootStore: RootStore) {
+    this.players = [];
+    this.rootStore = rootStore;
+    makeAutoObservable(this);
+  }
+
+  setPlayer(playerName: string, technology: Technologies) {
+    const newPlayer = new Player(playerName, technology);
+    this.players.push(newPlayer);
+  }
+
+  changeOrderOfPlayers(i: number) {
+    [this.players[i - 1], this.players[i]] = [this.players[i], this.players[i - 1]];
+  }
+  getCurrentPlayer() {
+    return this.players.find(
+      (_player, index) => index === (this.rootStore.gameStore.turnNumber - 1) % this.players.length,
+    );
```

Comment:
wouldn't it be easier to use just a this.player[modulo of turnnumber]? You are not using _player anyway so .find seems to be an overkill

Reply:
k